packages/suite-web/src/support/Router.tsx

```diff
@@ -144,23 +144,20 @@ const components: Record<PageName, LazyExoticComponent<ComponentType<any>>> = {
 };
 
 const AppRouter = () => (
-    // inititating strict mode higher would throw an error from react-helmet
-    // TODO: replace react-helmet with a maintained alternative
-    // strict mode is commented out because of its interplay with compose errors in send form
```

Comment:
See https://github.com/trezor/trezor-suite/issues/6694. Looks like it does not happen any more (don't know why), so we can close that issue when we merge this.